util-flywayinstaller/src/main/java/com/github/cafapi/util/flywayinstaller/Migrator.java

```diff
@@ -65,7 +65,7 @@ public static void migrate(
         final String password
     ) throws SQLException
     {
-        migrate(dbHost, dbPort, dbName, username, secretKeys, password, null, null);
+        migrate(dbHost, dbPort, dbName, username, secretKeys, password, null, null, null);
     }
 
     public static void migrate(
```

Comment:
Presumably we should deprecate this method as we would always want callers to supply the extra parameter?

```suggestion
    @Deprecated
    public static void migrate(
```